soda-core/src/soda_core/common/data_source_impl.py

```diff
@@ -214,3 +217,21 @@ def test_schema_exists(self, prefixes: list[str]) -> bool:
                 schema_exists = True
                 break
         return schema_exists
+
+    def verify_if_table_exists(self, prefixes: list[str], table_name: str) -> bool:
```

Comment:
would it be possible to use this in tests? Don't we check if table exists there as well?